### PR TITLE
This PR is a first stab at flexible mounting of volumes

### PR DESCRIPTION
**Pull request by donnydavis**
Please see issue 1012 for further details. 

https://github.com/heketi/heketi/issues/1012

Any help on getting this to work correctly would be great

### CODE REVIEW

File: executors/cmdexec/brick.go [hunk 1]
```diff
@@ -66,13 +66,13 @@ func (s *CmdExecutor) BrickCreate(host string,
 		fmt.Sprintf("mkfs.xfs -i size=512 -n size=8192 %v", devnode),
 
 		// Fstab
-		fmt.Sprintf("echo \"%v %v xfs rw,inode64,noatime,nouuid 1 2\" | tee -a %v > /dev/null ",
+		fmt.Sprintf("echo \"%v %v xfs %mountopts 1 2\" | tee -a %v > /dev/null ",
```
[the remark this answers is not in the document]
Comment (donnydavis) [reply executors/cmdexec/brick.go]: I agree. It makes more sense to have sane defaults with the ability to override. 

I would have just written it right the first time, but I am very new to go. So I do appreciate the help. I will get it running locally and then submit a new PR once it's not broken.

[the remark this answers is not in the document]
Comment (phlogistonjohn) [reply executors/cmdexec/brick.go]: No worries, I've only been using Go for less than 6 months myself. Thank you for working on this, the contributions are very much appreciated!

**Comment by obnoxxx** [comment]
@donnydavis Thanks for your contribution! Let us know if you need further assistance in getting the requested changes done. But we'll also happily let you do it at your own pace. :-)

**Comment by phlogistonjohn** [comment]
@donnydavis do you still intend to work on this PR?
If not would you object to the project maintainers adapting any of this work if we find it useful?

**Comment by ahmedsamer** [comment]
Can't use Heketi on production env until this PR (on any relevant one that fix mount options) is merged.

**Comment by donnydavis** [comment]
Sorry this one got lost in a job transition. Please feel free to run with it.

**Comment by wtownse** [comment]
working on this in PR 1679

**Comment by donnydavis** [comment]
Closing in favor of https://github.com/heketi/heketi/pull/1679

**Pull request closed without merging**